getting_started_vagrantbox.md

```diff
@@ -0,0 +1,85 @@
+## The goal of this guide 
+
+> :warning: Read the section `Description - what & why` in [README.md](/README.md) to get a quick introduction to what this repo is.
+
+The goal of this guide is to give an introduction to the [vagrant hashistack box](https://vagrantcloud.com/fredrikhgrelland/hashistack), its features, and how to use it.
+
+## What is the vagrant hashistack box
+The vagrant-hashistack box is a virtual machine built with [vagrant](https://www.vagrantup.com/), using the code from [this repository](https://github.com/fredrikhgrelland/vagrant-hashistack/). It aims to simulate a hashistack ecosystem, and with the help of built in switches and services it is able to be tweaked to a user's needs. The Norwegian Tax Administration is using this box as a part of their CI/CD pipeline to test terraform modules in their microservice architecture. 
+
+> :bulb: **Hashistack**, in current repository context, is a set of software products by [HashiCorp](https://www.hashicorp.com/).
+
+In the context of this template the box solves the problem of setting up a hashistack ecosystem, which is normally tedious and non-trivial, but with the box does it's done with one click.  
+
+### Vagrant box vs your local machine
+It's important to note that your local machine and the running vagrant box (from now on called virtual machine) are two separate entities. The virtual machine is available at the IP `10.0.3.10` and our local machine is available at the IP `127.0.0.1`, or alternatively `localhost`.
+
+TODO: generate PNG
+```mermaid
+graph TD
+  subgraph local machine
+    subgraph "virtual machine (10.0.3.10)"
+        Consul
+        Nomad
+        Vault
+        vagrant["/vagrant"]
+        MinIO
+        Other[Other softwares installed]
+    end
+    localhost["localhost (127.0.0.1)"]
+    pwd["/<path-to-vagrantfile>"]
+  end
+  %% links
+  Consul-->localhost
+  Nomad-->localhost
+  Vault-->localhost
+  vagrant-->pwd
+```
+
+### Services inside box
+The virtual machine has four services running:
+
+|Name|port|
+|:--|:-:|
+|Consul|8500|
+|Vault|8200|
+|Nomad|4646|
+|MinIO|9000|
+
+As mentioned the virtual machine can be reached at `10.0.3.10`, meaning the services will be available at `10.0.3.10:<port-number>`. For convenience Consul, Vault, and Nomad have all been forwarded to `localhost` as well, meaning they are available at `localhost:<port-number>`. MinIO has _not_ been forwarded.  
+
+### CLI-tools
+Nomad, Vault and Consul have their own CLI-tools to interact with the servers that are running. They all default to `localhost`, and the default ports mentioned above. This means you can download any of the binaries, and they will automatically be connected to the services running inside the virtual machine. Refer to [this section](#iteration-of-the-development-process) to see examples on how to use this.
```

Comment:
Is `iteration-of-the-development-process` a upcoming section?

Reply:
Nope, missing link, in README